_docs/datadestinations/google-analytics-costs.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: page
+section: datadestinations
+title: "Google Analytics Costs"
+order: 1
```

Comment:
I think we should use order like in data source section, where index page has order 0 and other has order 1.
It need for alphabet sorting.